### ISSUE TITLE
Phase mapper based on voltage levels

### ISSUE THREAD

**Issue by KapilDuwadi** [NREL-Distribution-Suites/shift#2]
### Preliminary Checks

- [X] I have searched existing issues for a duplicate feature request and I'm sure this is new.
- [X] I have read and followed [the docs](https://nrel-distribution-suites.github.io/shift) and still think this is missing.

### Description

It would be nice to have phase mapper utility provided within in shift where user can specify phases for different voltage levels.

**Comment by KapilDuwadi** [comment]
Needs #3 before we can work on this.
